libraries/anvill_passes/include/anvill/IndirectJumpPass.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <anvill/ABI.h>
+#include <llvm/IR/Instructions.h>
+#include <llvm/Pass.h>
+
+namespace anvill {
+namespace {
+bool isTargetInstrinsic(const llvm::CallInst *callinsn) {
+  if (const auto *callee = callinsn->getCalledFunction()) {
+    return callee->getName().equals(kAnvillSwitchCompleteFunc);
+  }
+
+  return false;
+}
+
+std::vector<llvm::CallInst *> getTargetCalls(llvm::Function &F) {
+  std::vector<llvm::CallInst *> calls;
+  for (auto &blk : F.getBasicBlockList()) {
```

Comment:
Two alternate ways of writing this:

```c++
for (auto &blk : *F) {
  for (auto &insn : blk) {
    ...
  }
}
```

The other is:

```c++
for (auto &insn : llvm::instructions(*F)) {
  ...
}
```

While it is common in the LLVM codebase to use capitalized variables like `F`, in this codebase use all lower-case variables names.